config/code_ocean.yml.example

```diff
@@ -68,6 +74,9 @@ production:
   <<: *default
   prometheus_exporter:
     enabled: true
+  content_moderation:
+    report_emails:
+      - report@example.com
```

Comment:
Didn't you want to add that config value to `development` instead of production?